### ISSUE TITLE
Proposal: disable markdownlint line-length check

### ISSUE THREAD

**Issue by bernhold** [betterscientificsoftware/bssw.io#1348]
We have a `.markdownlint.json` file at the top of the repo.  This is a configuration file for the markdownlint tool, which can be used in a GH action (but we’re not presently) and can also be used by some editors (e.g., VS Code).

One of the rules has to do with the maximum length of the lines in the file (MD013 aka line-length).  We have it configured to 80 characters.

**I propose to entirely disable this rule.**

Reasoning: Line length is a cosmetic issue.  It will not affect the interpretation of the text by a Markdown processor.  It is also not something that (this tool, at least) can automatically fix for us.  On the other hand, when I’m working with Markdown files in VS Code, my usual editor for BSSw.io, the linter flags every line after 80 characters with a squiggle underneath.  This obscures other, potentially important issues that the linter, spell checker, or other tools in the editor might catch.  I believe that it is far more useful to be able to catch real issues than to worry about cosmetic issues.

**Please discuss here, or at least provide a reaction to the proposal.**  I would like to make this change this week, if possible.

**Comment by markcmiller86** [comment]
> **I propose to entirely disable this rule.**

I agree 100%...lets disable it. @bartlettroscoe convinced me of the value in having lines=sentences a while back. It is now part of [VisIt's policies](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#about-line-length).
